clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -165,4 +165,17 @@
       </xs:annotation>
     </xs:attribute>
   </xs:attributeGroup>
+
+  <xs:element name="clustered-store" type="tc:clustered-store-type" substitutionGroup="eh:service-configuration"/>
```

Comment:
Do we want to call this element clustered-"store"? Store is more of an internal concept, right? How about having the config something like this:
`<consistency>STRONG</consistency>` and `<consistency>EVENTUAL</consistency>` ?